### PR TITLE
Lse01

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/dragino/lse01/uplink.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+function hexToBytes(hex) {
```
Comment (mariusgiger) [inline types/dragino/lse01/uplink.js:1]: use bits?

File: types/dragino/lse01/uplink.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  return bytes;
+}
+
+function consume(event) {
+  const payload = event.data.payload_hex;
+  const bytes = hexToBytes(payload);
+  const data = {};
+  const lifecycle = {};
+
+  let value = ((bytes[0] << 8) | bytes[1]) & 0x3fff;
```
Comment (mariusgiger) [inline types/dragino/lse01/uplink.js:14]: prevent reusing this variable for different calculations, rather use separate variables

File: types/globalsat/LT-20/uplink.js [hunk 3]
```diff
@@ -1,51 +1,56 @@
...
+  let tempHex = hex;
+  while (tempHex.length >= 2) {
+    hexArray.push(tempHex.substring(0, 2));
+    tempHex = tempHex.substring(2, tempHex.length);
   }
-  hexArray.reverse()
-  hex = hexArray.join('');
-
+  hexArray.reverse();
+  tempHex = hexArray.join("");
```
Comment (mariusgiger) [inline types/globalsat/LT-20/uplink.js:10]: why is hexArray assigned to tempHex?

File: types/dragino/lse01/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/dragino/lse01/uplink.js:1]: We don't have this function in the Bits library

File: types/dragino/lse01/default.schema.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "temperature": {
+      "title": "Temperature DS18B20",
+      "unit": "°C",
+      "type": "number",
+      "description": "The DS18B20 temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "soilHumidity": {
+      "title": "Water soil",
```
Comment (mariusgiger) [inline types/dragino/lse01/default.schema.json:16]: also change the title :)

File: types/dragino/lse01/default.schema.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "soilTemperature": {
+      "title": "Soil temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "soilConductivity": {
+      "title": "Conduct soil",
```
Comment (mariusgiger) [inline types/dragino/lse01/default.schema.json:32]: also change the title

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**